x-pack/platform/packages/shared/kbn-ai-assistant/src/hooks/use_knowledge_base.tsx

```diff
@@ -98,7 +106,8 @@ export function useKnowledgeBase(): UseKnowledgeBaseResult {
 
     const interval = setInterval(async () => {
       // re-fetch /status
-      await statusRequest.refresh();
+      statusRequest.refresh();
```

Comment:
@arturoliduena Do you think we can improve this code. It looks quite error prone the way we poll for status, and I'm pretty sure we can simplify and improve